### DISCUSSION TITLE
Other Overleaf Extended Editions

### DISCUSSION THREAD

**Discussion by svdvonde** [yu-i-i/overleaf-cep#9] [General]
Similar to this extended community edition, I have stumbled on alternative community extensions of Overleaf CE which are interesting to see the features that they have added/activated. The Overleaf Ruhr below even has github/gitlab sync.

-----
**Overleaf OIDC Edition**
 * Hosted at: https://tex.fachschaften.org/
 * Source code: https://github.com/Eisfunke/overleaf
 * Maintained by a council of active and former students of the TU Dortmund (to help students). 

Added features compared to Overleaf CE:
 * Auth via OpenID Connect

Has a custom hardcoded features, such as a login page in German.
 
-----

**Overleaf Ruhr**
 * Hosted at: https://tex.fs-its.rub.de/ and https://tex.cs.rub.de/login
 * Homepage: https://overleaf.lars.re/
 * Source code: https://gitlab.fachschaften.org/newton/overleaf-delta
 * Based on initial work of Overleaf OIDC Edition. 
 * Maintained by Lars, the person who developed it (see homepage)

Added features compared to Overleaf CE (list taken from here: https://overleaf.lars.re/):
 * Git Sync repository (GitHub / GitLab)
 * Real-time track changes / peer reviewing
 * Templates
 * Tags
 * Singe-Sign-On (via OpenID Connect)
 * Fully isolated compiling, enabling the secure usage of packages such as minted.
 * Full document history
 * Some reference search
 * Visual Editor
 * Symbol palette

 
Similar to Overleaf OIDC Edition, some features have been hardcoded for the german university where the application is deployed.

-----

**Overleaf LCPU**

 * Source code: https://github.com/lcpu-club/overleaf
 * Maintained by the Linux Club of Peking University

Added features:
 * Sandboxed compiles
 * Selecting docker build image for compiles
 * Review and comments
 * References autocomplete

**Comment by simon4477** [comment]
Thanks @svdvonde 

Could you please provide a step-by-step guide for a newbie on how to set it up with overleaf-toolkit? I have used overleaf toolkit, but I cannot find any documentation on how to setup the extended edition that you mentioned. Does it work with overleaf v5.2.1?

**Reply by svdvonde** [reply]
Hi. Depends on how newbie. I answered [here](https://github.com/yu-i-i/overleaf-cep/discussions/11#discussioncomment-11463011).

**Comment by yu-i-i** [comment]
I saw Overleaf Ruhr, but I couldn’t find either the templates I wanted or the reference search feature. I didn’t try Git sync, but I’m concerned it might be unreliable. Moreover, making it reliable is not an easy task, as noted  [here](https://github.com/overleaf/overleaf/issues/782#issuecomment-693336644). Anyway, as far as I understand, they promised to release their new version, possibly before Christmas. We’ll see...
